file-ballerina/src/file/filepath/windows_file_path.bal

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Check all other places as well

Reply:
We are just moving the files, not committing newly. We can keep the license year as it is.